packages/@adobe/spectrum-css-temp/components/table/index.css

```diff
@@ -196,6 +196,7 @@ svg.spectrum-Table-sortedIcon {
   line-height: calc(calc(var(--spectrum-table-cell-text-size) * var(--spectrum-table-cell-text-line-height)) - 1px);
   padding: var(--spectrum-table-cell-regular-padding-y) var(--spectrum-table-cell-padding-x);
 }
+.spectrum-Table--regular {}
```

Comment:
What is this for?

Reply:
Noticed we had a non-css modularized class name in the dom, it's this, it corresponds to https://github.com/adobe/react-spectrum/blob/main/packages/%40react-spectrum/table/src/TableView.tsx#L299
which can be one of three values, compact, spacious, or regular. the other two have classnames, so I just added the name so it'd also get modularized